src/main/java/com/github/manolo8/darkbot/gui/drawables/ConstantEntitiesDrawer.java

```diff
@@ -37,20 +36,20 @@ public void drawPortals(MapGraphics mg) {
         mg.setColor("portals");
 
         for (Portal portal : portals) {
-            mg.drawOvalCentered(portal, 12, false);
+            mg.drawOvalCentered(portal, 12f, false);
         }
     }
 
     public void drawBattleStations(MapGraphics mg) {
         for (BattleStation bs : battleStations) {
-            if (!(bs instanceof BattleStation.Module) && bs.getHullId() == 0) mg.setColor("meteroid");
+            if (bs instanceof BattleStation.Asteroid) mg.setColor("meteroid");
             else if (bs.getEntityInfo().isEnemy()) mg.setColor("enemies");
             else mg.setColor("allies");
 
             if (bs instanceof BattleStation.Module) {
-                mg.drawRectCentered(bs, 3, false);
+                mg.drawRectCentered(bs, 3f, false);
```

Comment:
```suggestion
                mg.drawRectCentered(bs, 3d, false);
```
Please change the float params to doubles, the idea of the change is for them to use the double-based method after all